frontend/src/pages/browse-page/browse-page.tsx

```diff
@@ -69,14 +75,14 @@ const BrowsePage: FC = () => {
         })}
       </div>
       <div className={styles['browse-page-video-container']}>
-        {videoData.dataStatus === DataStatus.PENDING ? generateHistorySkeletons(isLightTheme) : null}
+        {videoData.dataStatus === DataStatus.PENDING && generateBrowsePageSkeleton(isLightTheme, lastListLength)}
         {popularVideos.list.map((video) => {
           return <VideoCardMain key={video.id} video={video} isLightTheme={isLightTheme} />;
         })}
         <div ref={sentryRef}>
-          {videoData.dataStatus === DataStatus.PENDING && popularVideos.list.length > 0 ? (
-            <Loader hCentered={true} vCentered={true} spinnerSize={'md'} />
-          ) : null}
+          {videoData.dataStatus === DataStatus.PENDING && popularVideos.list.length > 0
```

Comment:
You can wrap the expression in parentheses and you won't need to use the ternary operator

Reply:
done